XamarinCommunityToolkit/Effects/EntryFormattedTextEffect.android.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.ComponentModel;
+using Android.Text;
+using Android.Text.Style;
+using Android.Widget;
+using Xamarin.CommunityToolkit.Android.Effects;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ResolutionGroupName(nameof(Xamarin.CommunityToolkit))]
+[assembly: ExportEffect(typeof(EntryFormattedTextEffect), nameof(EntryFormattedTextEffect))]
+
+namespace Xamarin.CommunityToolkit.Android.Effects
+{
+	/// <summary>
+	/// The FormattedText <see cref="PlatformEffect"/> specific
+	/// to <see cref="Entry"/> controls.
+	/// </summary>
+	public class EntryFormattedTextEffect : PlatformEffect, IDisposable
+	{
+		bool isDisposed;
+
+		/// <summary>
+		/// Gets the native <see cref="FormsEditText"/> control.
+		/// </summary>
+		public new FormsEditText Control => (FormsEditText)base.Control;
+
+		/// <summary>
+		/// Gets the Xamarin.Forms <see cref="Entry"/> element.
+		/// </summary>
+		public new Entry Element => (Entry)base.Element;
+
+		/// <inheritdoc />
+		protected override void OnAttached()
+		{
+			UpdateFormattedText();
+			Control.TextChanged += OnTextChanged;
+		}
+
+		/// <inheritdoc />
+		protected override void OnDetached()
+		{
+		}
+
+		/// <inheritdoc />
+		protected override void OnElementPropertyChanged(PropertyChangedEventArgs args)
+		{
+			base.OnElementPropertyChanged(args);
+
+			if (args.PropertyName == FormattedTextEffect.FormattedTextProperty.PropertyName)
+			{
+				UpdateFormattedText();
```

Comment:
When we have just one call, we don't use brackets... I still working to make our analyzer better, looks like editorconfig is being ignored by VS.

Reply:
My mistake, I thought stylecop told me otherwise.

I'll update this, trying to make sure I'm 100% compliant with the coding standard

Reply:
Don't worry, the stylecop doesn't have a definition for this case (I didn't find so far), this one should be `editorconfig` responsibility, but he has not been working well.

Reply:
I'm going to see if I can find the stylecop setting for this. I've seen it in other projects I've worked on. I'll DM you about it if I find it

Reply:
This has been fixed in the latest code push